src/Compilers/Core/Portable/Symbols/ITypeParameterSymbol.cs

```diff
@@ -68,7 +71,10 @@ public interface ITypeParameterSymbol : ITypeSymbol
         /// </summary>
         ImmutableArray<ITypeSymbol> ConstraintTypes { get; }
 
-        // PROTOTYPE(nullable-api): Document
+        /// <summary>
+        /// The top-level nullabilities that were directly specified as constraints on the
+        /// type parameter.
```

Comment:
Perhaps "of the <see cref="ConstraintTypes"/>." so it's clear these from the constraint types directly.